csharp/Platform.Collections/Segments/Walkers/AllSegmentsWalkerBase.cs

```diff
@@ -3,19 +3,16 @@
 namespace Platform.Collections.Segments.Walkers
 {
     /// <summary>
-    /// <para>
-    /// Represents the all segments walker base.
-    /// </para>
-    /// <para></para>
+    /// <para>Represents the base abstract class for walkers on all elements.</para>
+    /// <para>Представляет базовый класс для проходчиков по всем элементам.</para>
     /// </summary>
     public abstract class AllSegmentsWalkerBase
     {
         /// <summary>
-        /// <para>
-        /// The default minimum string segment length.
-        /// </para>
-        /// <para></para>
+        /// <para>Gets a default minimum length of the string segment.</para> 
```

Comment:
A read-only field that represents ...
... the default minimum sequence segment length.